src/hooks/useAccessibility.ts

```diff
@@ -15,13 +15,15 @@ export const useAccessibility = (
     const triggerEvent: string = ACCESSIBILITY_EVENTS_MAP[trigger];
     const handleAccessibility = useCallback(
         (e) => {
-            if (ref.current?.contains?.(e.target)) {
-                if (triggerEvent != 'keydown' || e.key == 'Enter') {
-                    return onEnter?.();
+            if (ref.current) {
```

Comment:
The handler is specific, but the hook is generalized. You should create a switch by key using `shared/utilities/eventKeys.ts` so this may be extended without being refactored.

Reply:
we are not changing the existing functionality reg. the events.